walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -1,39 +1,40 @@
+:walkthrough: Integrating message-oriented middleware with a RESTful API using AMQ
+:messaging-service: Red Hat AMQ
 :fuse-version: 7.2
+:AMQ-ProductLongName: Red Hat AMQ
+:AMQ-BrokerVersion: 7.2
+:Fuse-prodnamefull: Red Hat Fuse Online
+:node-url: http://frontend-node-app.apps.city.openshiftworkshop.com/
+:spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
+:fuse-url: https://eval.apps.city.openshiftworkshop.com/
+
 
 
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id='integrating-eventdriven-and-apidriven-applications']
-// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules-{context}'].
 
-// Book Title
 = {walkthrough}
 
+// word count range that fits best is 15-22, with 20 really being the sweet spot. Character count for that space would be 100-125
+This walkthrough demonstrates a fault tolerant integration pattern that integrates a message driven architecture with a RESTful API architecture.
 
-//If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
-:context: integrating-eventdriven-and-apidriven-applications
-// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
-
-//include::master-1-intro.adoc[ ]
+Message-oriented middleware (MOM)::
+MOM allows you implement an asynchronous, routable and transformable message queue.
 
-// word count range that fits best is 15-22, with 20 really being the sweet spot. Character count for that space would be 100-125
-Build a simple integration that connects a message queue with an API.
+API-centric architecture::
+Typically, a developer working on a back-end application implements a synchronous RESTful API to allow communication from other applications.
 
-In a multi-application system, you often find:
+You may identify business value, as part of an application modernization effort, in integrating a message queue with an API.
+This is not trivial because of the conflicting underlying timing models, and the data may require transformation to be usable.
 
-* communication between the applications can be difficult due to differing technologies
-* custom solutions can be difficult to create and maintain
-* custom solutions require custom monitoring
-* understanding the relationship between the applications requires deep technical insight to the overall architecture
+In this walkthrough, you create a simple and intuitive integration, with built-in monitoring, to enable communication between two applications:
 
-A more structured approach involves:
+* A link:{route-frontend-host}[Node.js webapp, window="_blank"] provides a method to create messages, in this example, a new inventory item.
```

Comment:
@finp Lets remove these links as the resources won't be created until the user clicks 'Get Started' on the overview page